release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -294,6 +294,13 @@ Because the generated certificates contain wildcard subjects for headless servic
 
 For more information, see xref:../security/certificates/service-serving-certificate.adoc#add-service-certificate_service-serving-certificate[Add a service certificate].
 
+[id="ocp-4-8-security-oc-compliance-plug-in"]
+==== The oc-compliance Plug-in is now available
```

Comment:
Should Plug-in be plug-in? (multiple instances)

Reply:
This is a special case because the name of the Operator is a proper noun